.travis.yml

```diff
@@ -7,17 +7,22 @@ julia:
   - 1
   - nightly
 notifications:
-  email: false
+    email:
+    recipients:
+      - nightly-dev@invenia.ca
+    on_success: never
+    on_failure: always
+    if: type = cron
 matrix:
   allow_failures:
-    - julia: nightly
+  - julia: nightly
 branches:
   only:
-    - master
-    - staging
-    - trying
-    - /^v[0-9]+\.[0-9]+\.[0-9]+$/  # version tags
+  - master
+  - staging
+  - trying
+  - "/^v[0-9]+\\.[0-9]+\\.[0-9]+$/"
```

Comment:
Maybe add back the comment?

Reply:
Can do! This just happened when running:

> travis pro encrypt {creds} --add

It likes to re-format the file